types/src/front/lib/api/assistant/actions/database_query.ts

```diff
@@ -0,0 +1,20 @@
+import { DatabaseQueryActionType } from "front/assistant/actions/database_query";
+
+export type DatabaseQueryRunErrorEvent = {
```

Comment:
Remove the `Run` the action is `DatabaseQuery`  (this run was copied from the action `DustAppRun`)